src/Autoloading/BootstrapFilesIncluder.php

```diff
@@ -69,8 +74,13 @@ private function resolveAbsoluteBootstrapFilePaths(array $extensionConfigFiles):
         $absoluteBootstrapFilePaths = [];
 
         foreach ($extensionConfigFiles as $extensionConfigFile) {
-            $extensionConfigContents = Neon::decodeFile($extensionConfigFile);
+            if (!array_key_exists($extensionConfigFile, $this->configCache)) {
+                $extensionConfigContents = Neon::decodeFile($extensionConfigFile);
 
+                $this->configCache[$extensionConfigFile] = $extensionConfigContents;
+            } else {
+                $extensionConfigContents = $this->configCache[$extensionConfigFile];
+            }
```

Comment:
I think this could be simplified to 

```php
$extensionConfigContents = $this->configCache[$extensionConfigFile] ??= Neon::decodeFile($extensionConfigFile);
```